website/content/docs/drivers/remote/ecs.mdx

```diff
@@ -116,7 +116,8 @@ job "nomad-ecs-demo" {
       - `subnets` - The subnets associated with the task or service.
 
 [cluster]: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/clusters.html
-[demo]: https://github.com/hashicorp/nomad-driver-ecs/blob/main/demo/terraform/ecs.tf
-[download]: https://releases.hashicorp.com/nomad-ecs-driver/
+[ecs]: https://aws.amazon.com/ecs/
+[demo]: https://github.com/hashicorp/nomad-driver-ecs/tree/main/demo
```

Comment:
This link is still broken where it's used. I think https://github.com/hashicorp/nomad/pull/10627/files#diff-2611dcb25a9897a1dce30ba8b53eb17c0bb87ff5e7a438f14b1d55e3d42477c7R63-R64 needs to be:

```
Nomad ECS tasks must first be defined for the ECS cluster. See the [Nomad ECS
driver demo][demo] task for an example ECS task provisioned by Terraform.
```

Reply:
@karl-cardenas-coding I did a touchup to get that link sorted.

Reply:
@angrycub thank you :)